.openpublishing.redirection.json

```
@@ -1,3811 +1,3798 @@
-        },
-        {
-            "source_path": "docs/csharp/programming-guide/linq-query-expressions/how-to-perform-custom-join-operations.md",
-            "redirect_url": "/dotnet/csharp/linq/perform-custom-join-operations"
-        },
-        {
-            "source_path": "docs/csharp/programming-guide/linq-query-expressions/how-to-perform-grouped-joins.md",
-            "redirect_url": "/dotnet/csharp/linq/perform-grouped-joins"
-        },
-        {
-            "source_path": "docs/csharp/programming-guide/linq-query-expressions/how-to-perform-inner-joins.md",
-            "redirect_url": "/dotnet/csharp/linq/perform-inner-joins"
-        },
-        {
-            "source_path": "docs/csharp/programming-guide/linq-query-expressions/how-to-perform-left-outer-joins.md",
-            "redirect_url": "/dotnet/csharp/linq/perform-left-outer-joins"
-        },
-        {
-            "source_path": "docs/csharp/programming-guide/linq-query-expressions/how-to-query-a-collection-of-objects.md",
-            "redirect_url": "/dotnet/csharp/linq/query-a-collection-of-objects"
-        },
-        {
-            "source_path": "docs/csharp/programming-guide/linq-query-expressions/how-to-return-a-query-from-a-method.md",
-            "redirect_url": "/dotnet/csharp/linq/return-a-query-from-a-method"
-        },
-        {
-            "source_path": "docs/csharp/programming-guide/linq-query-expressions/how-to-store-the-results-of-a-query-in-memory.md",
-            "redirect_url": "/dotnet/csharp/linq/store-the-results-of-a-query-in-memory"
-        },
-        {
-            "source_path": "docs/csharp/programming-guide/linq-query-expressions/how-to-write-linq-queries.md",
-            "redirect_url": "/dotnet/csharp/linq/write-linq-queries"
-        },
-        {
-            "source_path": "docs/csharp/programming-guide/linq-query-expressions/index.md",
-            "redirect_url": "/dotnet/csharp/linq/index"
-        },
-        {
-            "source_path": "docs/csharp/programming-guide/linq-query-expressions/query-expression-basics.md",
-            "redirect_url": "/dotnet/csharp/linq/query-expression-basics"
-        },
-        {
-            "source_path": "docs/csharp/programming-guide/main-and-command-args/how-to-access-command-line-arguments-using-foreach.md",
-            "redirect_url": "/dotnet/csharp/programming-guide/arrays/using-foreach-with-arrays"
-        },
-        {
-            "source_path": "docs/csharp/programming-guide/main-and-command-args/main-and-command-line-arguments.md",
-            "redirect_url": "/dotnet/csharp/programming-guide/main-and-command-args/index"
-        },
-        {
-            "source_path": "docs/csharp/programming-guide/namespaces/how-to-use-the-global-namespace-alias.md",
-            "redirect_url": "/dotnet/csharp/language-reference/operators/namespace-alias-qualifier"
-        },
-        {
-            "source_path": "docs/
```

Shouldn't we keep this doc id redirect?

```suggestion
      "redirect_document_id": true
      "redirect_url": "/dotnet/desktop-wpf/themes/how-to-create-apply-template"
```

---

But now you've already published the new article without the proper redirect. So transferring the BI data now could mess things up, I think.

---

Because when you do that in sync, the new article gets the id of the old one when published. The new one has been live for a few months now so it has its own BI data now, if that makes sense.